source/fundamentals/indexes.txt

```diff
@@ -242,9 +244,20 @@ language as an option when creating the index.
    Text indexes differ from the more powerful :atlas:`Atlas full text search indexes </atlas-search/>` Atlas users
    should use Atlas search.
 
-.. warning::
+.. warning:: Restriction on one text index per collection. 
```

Comment:
Since this seems like you're describing a feature in the content, I'm not sure warning is the best representation. Indexing multiple fields for a text index seems more like fundamental learning material than something that may directly generate issues for the user. Maybe you can separate out what you're warning the reader about from the content about what you can do with a compound text index?

Also, I wonder if this belongs here or after showing the reader what a single field text search index looks like. Is there any reason we should show this example before that?

Also, for admonitions like "warning" or "info", I would prefer a complete sentence in the title, e.g.
"A collection can only contain one text index."

Reply:
Agreed, I will keep the warning to state what you can't do and show how to do it right after